### PR TITLE
Fix getMyMatch, add setMatchResult

### PR DESCRIPTION
**Pull request by gordonshieh**
@landalex please review! :)

### CODE REVIEW

File: gordonshieh94/rockpaperscissors/utils/redis.js [hunk 1]
```diff
@@ -26,11 +26,26 @@ async function joinQueue(name) {
 
 async function getMyMatch(name) {
     let keys = await client.keys();
-    let myMatch = keys.find(x => x.startsWith(name))
-    return myMatch
+    let myMatch = keys.find(x => x.startsWith(name)).split('_')[1];
```
Comment (landalex) [inline gordonshieh94/rockpaperscissors/utils/redis.js]: Bad variable name

File: gordonshieh94/rockpaperscissors/utils/redis.js [hunk 2]
```diff
@@ -26,11 +26,26 @@ async function joinQueue(name) {
...
+    return myMatch;
+}
+
+async function setMatchResult(name, match, result) {
+    await client.set(`${name}_${match}`, result);
+    let keys = await client.keys();
+    let otherPlayersMatch = keys.find(x => !x.startsWith(name) && x.endsWith(match));
```
Comment (landalex) [inline gordonshieh94/rockpaperscissors/utils/redis.js]: Really?

File: gordonshieh94/rockpaperscissors/utils/redis.js [hunk 3]
```diff
@@ -26,11 +26,26 @@ async function joinQueue(name) {
 
 async function getMyMatch(name) {
```
Comment (landalex) [inline gordonshieh94/rockpaperscissors/utils/redis.js:27]: Give me a break

**Review by landalex — COMMENTED** [review]
Get this out of my sight

File: gordonshieh94/rockpaperscissors/utils/redis.js (hunk 2 above)
Comment (gordonshieh) [reply gordonshieh94/rockpaperscissors/utils/redis.js]: k
fixed.

**Pull request merged**